iothub/device/src/Authentication/Security/SharedAccessSignatureBuilder.cs

```diff
@@ -16,6 +16,7 @@ namespace Microsoft.Azure.Devices.Client
     /// </summary>
     internal sealed class SharedAccessSignatureBuilder
     {
+        private const int DEFAULT_TOKEN_EXPIRY_MINS = 60;
```

Comment:
The SDK should already be maintaining a default ttl reference somewhere. Reuse that instead of adding a new const again.